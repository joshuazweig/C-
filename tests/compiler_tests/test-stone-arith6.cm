int main () {
    stone s1;
    stone s2;
    stone s3;
    stone s4;

    s1 = "4261379284712349123746192345";
    s2 = "2348178942317623416767421766";
    s3 = s1 * s2;
    s4 = s2 * s1;

    print_stone(s1);
    print_stone(s2);
    print_stone(s3);
    print_stone(s4);
}
