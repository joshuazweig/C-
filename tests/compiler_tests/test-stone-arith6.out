4261379284712349123746192345
2348178942317623416767421766
10006481101590074588285209139549239086165805985175581270
10006481101590074588285209139549239086165805985175581270
